packages/web-components/src/components/va-minimal-header/va-minimal-header.scss

```diff
@@ -19,6 +19,10 @@
   min-height: 30px;
   min-width: 48px;
 
+  &:focus {
```

Comment:
This needs to be added to the `a` tag instead of the image. When tabbing through the gold outline doesn't show because the image is not the focusable element.